server.js

```diff
@@ -651,20 +651,59 @@ proxyForAuditing.on('error', function(err, req, res) {
   serverUtils.serverError(JSON.stringify(err), req, res);
 });
 
-var couchDbVersionCheck = function(callback) {
-  db.getCouchDbVersion(function(err, version) {
-    console.log('CouchDB Version:', version);
+const nodeVersionCheck = callback => {
+  try {
+    console.log('Node Version:', process.version);
+
+    const version = process.versions.node.match(/(\d)+\.(\d)+\.(\d)+/)[1];
```

Comment:
Hey you're right, that is dodgy. I have no idea why I did it that way! (It's also done that way in Sentinel, I'll change Sentinel's to whatever I do here as well)